compiler-rt/lib/builtins/fp_lib.h

```diff
@@ -346,15 +346,6 @@ static __inline fp_t __compiler_rt_logbf(fp_t x) {
 static __inline fp_t __compiler_rt_scalbnf(fp_t x, int y) {
   return __compiler_rt_scalbnX(x, y);
 }
-static __inline fp_t __compiler_rt_fmaxf(fp_t x, fp_t y) {
```

Comment:
@alexander-shaposhnikov The same issue exists for doubles just a few lines below, are you planning to change that too?

Reply:
@arichardson Yeah, it would be good to change that too. I'm currently out of my bandwidth, but it would be great to clean this up as well.